### PR TITLE
Import adjustments

### LINKED ISSUE #6667: Import: allow to only import DB or configuration
**Is your feature request related to a problem? Please describe.**
When you want to migrate an existing setup to a new one, export/import mechanism is used the first time to have identical configuration and data. 

Once all tests have been performed on new setup, you may want to re-import **only** DB or configuration from old setup.

**Describe the solution you'd like**
Be able to import only DB or configuration from an export.

### PR DESCRIPTION
**Pull request by nqb**
# Description
- Add option to import only database or configuration
- Install/Uninstall Mariabackup if necessary to import DB dump made using Xtrabackup/Mariabackup

# Impacts
Import mechanism

# Issue
fixes #6667 

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* Allow to import only DB or configuration during import

### CODE REVIEW

**Comment by nqb** [comment]
Notable change:
- I move `restore_certificates` call in `import_config` block. Previously, this function was called after restart of services and I don't see a valid reason for that.

**Comment by nqb** [comment]
TODO:
- [x] update docs to remove install/uninstall mariabackup stuff if everybody is happy with code

**Comment by nqb** [comment]
I tested carefully following scenarios:
- [X] FULL Import 10.3 (mariabackup)
- [X] DB Import 10.3 (mariabackup)
- [X] Config Import 10.3 (mariabackup)
- [X] FULL Import 12.0 (mariabackup)
- [X] DB Import 12.0 (mariabackup)
- [X] Config Import 12.0 (mariabackup)

on Debian 11, running PF 12.0 and Rocky 8, running on `devel`:

**Comment by nqb** [comment]
It could be interested to add a flag to skip `adjust_configuration` function. 

Purpose: when you reimport configuration of a current cluster on a new built cluster, you don't want settings in `pf.conf` to be reset

**Comment by julsemaan** [comment]
The changes look good. Only thing would be to add the note in the doc regarding the mariadb-backup removal so it doesn't become a silent 'gotcha'

**Comment by nqb** [comment]
@julsemaan, I did what you requested and also add the ability to skip configuration adjustments.

I don't plan to add other things. For me, this PR is ready to use.

**Comment by nqb** [comment]
I created a patch to apply it on maintenance/11.2. I think it could be interesting for customers/users who want to stay on 11.2

File: docs/installation/export_import_mechanism.asciidoc [hunk 1]
```diff
@@ -113,43 +113,13 @@ the server.
...
-yum install MariaDB-backup --enablerepo=packetfence
-----
-
-.Debian based systems
-[source,bash]
-----
-apt install mariadb-backup
-----
+Consequently, if you installed Mariabackup **before** running
+import script, you should ensure that Mariabackup is still installed at end of import.
```
Comment (julsemaan) [inline docs/installation/export_import_mechanism.asciidoc]:
```suggestion
the import script, you should ensure that Mariabackup is still installed at end of import.
```

File: docs/installation/export_import_mechanism.asciidoc [hunk 2]
```diff
@@ -170,6 +140,8 @@ Completed import of the database and the configuration! Complete any necessary a
 
 If that's not the case, check the output above to understand why the process failed.
 
+If you experimented any issues during import, you can run it again.
```
Comment (julsemaan) [inline docs/installation/export_import_mechanism.asciidoc]:
```suggestion
If you experience any issues during import, you can run it again.
```

File: docs/installation/export_import_mechanism.asciidoc [hunk 3]
```diff
@@ -180,21 +152,4 @@ If your previous installation was a cluster, some steps may not be necessary
...
-----
-
-.Debian based systems
-[source,bash]
-----
-rm /usr/local/bin/mariabackup
-rm /usr/local/bin/mbstream
-----
-
+WARNING: if you installed Mariabackup before running import, it's possible that you need to reinstall it.
```
Comment (julsemaan) [inline docs/installation/export_import_mechanism.asciidoc]:
```suggestion
WARNING: if you installed Mariabackup before running the import, it's possible that you need to reinstall it.
```

**Comment by nqb** [comment]
Thanks @julsemaan, I merged your suggestions in one commit.

**Pull request merged**